jena/JosekiExtensions/src/com/bbn/parliament/jena/joseki/bridge/tracker/TrackableQuery.java

```diff
@@ -25,6 +27,15 @@
  */
 public class TrackableQuery extends Trackable {
 	private static Logger _log = LoggerFactory.getLogger(TrackableQuery.class);
+	
+	private static final Long TIMEOUT_DURATION;
+	private static final TimeUnit TIMEOUT_UNIT;
+
+	static {
+		Config config = Config.readFromFile();
+		TIMEOUT_DURATION = config.m_timeoutDuration;
+		TIMEOUT_UNIT = TimeUnit.valueOf(config.m_timeoutUnit);
```

Comment:
We should give some thought to what happens if (a) the configuration file is malformed or (b) the timeout unit string doesn't correspond to a member of TimeUnit.  As it stands here, I suspect that the process will die a nasty death without an error message.